### PR TITLE
fix: Don't set passwordless registration for users

### LINKED ISSUE #23: Remove initialize passwordlss from zitadel user create api call
### Description
This call breaks the "connect account experience". Let's remove it as it is a feature people will not use.

### PR DESCRIPTION
**Pull request by tlater-famedly**
This is *not* enabling users to do their registration without a password, it instead configures setting up registration with authenticators.

Fixes #23 and #4

### CODE REVIEW

**Comment by tlater-famedly** [comment]
Well, that didn't work, looks like we'll need #4 first

**Comment by tlater-famedly** [comment]
Keeping this as a draft until we confirm that this actually works as expected; I fear that this will force users to use SSO.

**Comment by tlater-famedly** [comment]
This now depends on famedly/zitadel-rust-client#9

File: src/lib.rs [hunk 1]
```diff
@@ -11,11 +11,15 @@ mod config;
...
 mod zitadel;
 
-pub use config::{AttributeMapping, Config};
+pub use config::{AttributeMapping, Config, FeatureFlag};
 use zitadel::Zitadel;
 
 /// Run the sync
 pub async fn sync_ldap_users_to_zitadel(config: Config) -> Result<()> {
+	if !config.feature_flags.contains(&FeatureFlag::SsoLogin) {
+		anyhow::bail!("Non-SSO configuration is currently not supported");
```
Comment (sirewix) [inline src/lib.rs:20]: Is it going to be supported at all? Otherwise having this flag is pointless

Comment (tlater-famedly) [reply src/lib.rs:20]: Yes, but we need to wait for zitadel to add support for it. See the commit message.

**Review by sirewix — APPROVED** [review]

**Pull request merged**